sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/DirectoryAPITest.groovy

```diff
@@ -434,6 +444,786 @@ class DirectoryAPITest extends APISpec {
         thrown(DataLakeStorageException)
     }
 
+    def "Set ACL recursive min"() {
```

Comment:
Can we have some tests on File? Maybe just a min test?

Reply:
Oh just read your other comment

Reply:
Yea they should be there now